skbio/metadata/base.py

```diff
@@ -0,0 +1,60 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2023, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
```

Comment:
Let's not forget to remove these headers.  We'll want to replace this with 

```
# ----------------------------------------------------------------------------
# Copyright (c) 2013--, scikit-bio development team.
#
# Distributed under the terms of the Modified BSD License.
#
# The full license is in the file LICENSE.txt, distributed with this software.
# ----------------------------------------------------------------------------
```

Reply:
I'm not totally sure since this is a backport, and the code originated in QIIME 2, and the attribution needs to be clear.

Reply:
Maybe we should keep the original headers then?

Reply:
@wasade @mortonjt In other similar situations in scikit-bio, the external sources are acknowledged in place and the external license files are added to scikit-bio, whereas the script header is scikit-bio's (as Jamie suggested). Here is an [example](https://github.com/scikit-bio/scikit-bio/blob/df931a0f518f105cb648b29fe38b2a114b5ece96/skbio/util/_misc.py#L204). And [another](https://github.com/scikit-bio/scikit-bio/blob/df931a0f518f105cb648b29fe38b2a114b5ece96/.coveragerc#L1).